snap/snapcraft.yaml

```
@@ -6,6 +6,13 @@ base: core18
 grade: stable
 confinement: strict
 
+architectures:
+  - build-on: amd64
+  - build-on: i386
+  - build-on: arm64
+  - build-on: armhf
+  - build-on: ppc64el
```

Will it work this way?
```suggestion
architectures: [amd64, i386, arm64, armhf, ppc64el]
```

---

No. Look at example 6:
https://snapcraft.io/docs/architectures

---

Okay.